.github/workflows/continuous_integration.yml

```diff
@@ -0,0 +1,47 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - develop
+      - master
+  pull_request:
+
+jobs:
+  build:
+    name: build
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.6", "3.7", "3.8"]
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Cache dependencies
+        uses: actions/cache@v2
+        with:
+          path: ~/.cache/pip
+          key: pip-${{ hashFiles('setup.cfg') }}-${{ hashFiles('tox.ini') }}
+          restore-keys: pip-
+
+      - name: Install packages
+        run: pip install flake8
+
+      - name: Lint
+        run: flake8
```

Comment:
We may want to move these dependencies into `setup.cfg` from a packaging semantics standpoint, like we do [here](https://github.com/azavea/django-ecsmanage/blob/develop/setup.cfg#L34-L37). But I think that's an unrelated task and should be part of a separate PR if we decide to do so. This works fine as-is.

Reply:
Good call! I opened https://github.com/azavea/python-omgeo/issues/65